examples/advanced-transform-2/advanced-transform-2.go

```diff
@@ -0,0 +1,80 @@
+package main
```

Comment:
Please call this example something nicer than "advanced-transform-2". How about "git-like-store"?

Reply:
yes, it deserves better.